src/services/AnalysisService.ts

```diff
@@ -301,65 +301,87 @@ class AnalysisService {
 
     const isGeneratedScanType = GeneratedScanTypes.includes(scanType);
 
-    const vulnerabilities = isGeneratedScanType
-      ? `(${StringUtilities.pluralizeTemplate(
-          scanStatus.issues?.Vulnerability?.count ?? 0,
-          "vulnerability",
-          "vulnerabilities",
-        )}) `
-      : "";
-
-    const codeIssues =
-      scanType === ScanType.SAST
-        ? `(${StringUtilities.pluralizeTemplate(
-            scanStatus.issues?.Sast?.count ?? 0,
-            "code issue",
-          )}) `
-        : "";
-
-    const webVulnerabilities =
-      scanType === ScanType.DAST
-        ? `(${StringUtilities.pluralizeTemplate(
-            scanStatus.issues?.Dast?.count ?? 0,
-            "web vulnerability",
-            "web vulnerabilities",
-          )}) `
-        : "";
-
-    const violations = isGeneratedScanType
-      ? `(${StringUtilities.pluralizeTemplate(
-          scanStatus.issues?.Violation?.count ?? 0,
-          "violation",
-        )}) `
-      : "";
-
-    const substitutions = isGeneratedScanType
-      ? `(${StringUtilities.pluralizeTemplate(
-          scanStatus.issues?.DependencySubstitution?.count ?? 0,
-          "dependency substitution",
-        )}) `
-      : "";
-
-    const typos = isGeneratedScanType
-      ? `(${StringUtilities.pluralizeTemplate(
-          scanStatus.issues?.DependencyTypo?.count ?? 0,
-          "dependency typo",
-        )}) `
-      : "";
-
-    const unknownPackages = isGeneratedScanType
-      ? `(${StringUtilities.pluralizeTemplate(
-          scanStatus.issues?.UnknownPackage?.count ?? 0,
-          "unknown package",
-        )}) `
-      : "";
-
+    soosLogger.always("".padEnd(25, "-"));
     soosLogger.always(
```

Comment:
made this ticket today - https://soos.atlassian.net/browse/PA-14419 - wanna stick a public method in there that dumps all that as a string (newline seperator instead of soosLogger.always for each) given a scan status

getFinalScanStatusMessage(scanStatus: IScanStatus)

existing method would call it e.g.
soosLogger.always(getFinalScanStatusMessage(scanStatus));

then we can call it directly in the vscode and map to a window message

Reply:
will have to double check the output on the pass-through integrations as well (e.g. dast) to ensure that formatting makes it through ok

for the vscode version might also need to make the '-' an opt-out param on that method